mmengine/dist/dist.py

```diff
@@ -504,7 +508,8 @@ def broadcast_object_list(data: List[Any],
         if group is None:
             group = get_default_group()
 
-        if digit_version(TORCH_VERSION) >= digit_version('1.8.0'):
+        if digit_version(TORCH_VERSION) >= digit_version(
+                '1.8.0') and not is_npu_available():
```

Comment:
If NPU does not support `broadcast_object_list`, adding an AssertionError may be proper.

Reply:
using _broadcast_object_list instead